doc/sphinx/source/notes_7_0.rst

```diff
@@ -2,10 +2,27 @@
 Release Notes for PoCL 6.1
 **************************
 
+* removed support for building tests & examples with OpenCL < 3.0
+
+* Support Khronos ICD if the version >= 3.0
+
+* CMake 3.15 is now required to build PoCL
+
+* New env variable POCL_MAX_COMPUTE_UNITS. This is similar to
+  existing POCL_CPU_MAX_CU_COUNT, but is not driver-specific,
+  hence can be used to limit the CU count of all drivers that
+  support limiting CU count. If both are specified, the driver
+  specific variable takes precedence.
+
+* Implemented OpenSSF best practices.
+
 ===========================
 Driver-specific features
 ===========================
 
+context/mem destructor callbacks, event callbacks, were moved
```

Comment:
Sentences should start with a capital letter.

Reply:
fixed